crush-main-board.kicad_pcb

```diff
@@ -3830,12 +3839,18 @@
   (gr_text "38" (at 20.5232 38.7604) (layer "F.SilkS") (tstamp 51756af9-a586-4f34-a94c-bebe7bfefad8)
     (effects (font (size 1 1) (thickness 0.15)) (justify left bottom))
   )
+  (gr_text "V" (at 64.262 129.6416) (layer "F.SilkS") (tstamp 530ecd37-e019-46ba-a37c-50e8ac17ab69)
```

Comment:
I think it might be more intuitive to have the +voltage line be labeled + instead of V, but otherwise LGTM

Reply:
Fixed!